mlir/test/Conversion/ArmSMEToLLVM/unsupported.mlir

```diff
@@ -1,4 +1,5 @@
-// RUN: mlir-opt %s -allocate-arm-sme-tiles -convert-arm-sme-to-llvm -split-input-file -allow-unregistered-dialect -verify-diagnostics
+// RUN: mlir-opt %s --pass-pipeline="builtin.module(func.func(convert-arm-sme-to-llvm))" \
+// RUN: -split-input-file -allow-unregistered-dialect -verify-diagnostics
```

Comment:
nit: replace `prevent.dce` with `test.some_use` and remove `-allow-unregistered-dialect` like you've done elsewhere? `-split-input-file` can also be removed.
```suggestion
// RUN: -verify-diagnostics
```